twine/settings.py

```diff
@@ -223,6 +229,13 @@ def register_argparse_arguments(parser):
             action="store_true",
             help="Show verbose output."
         )
+        parser.add_argument(
+            "-v", "--verify",
+            default=True,
+            required=False,
+            action="store_false",
+            help="Upload with or without SSL verification."
```

Comment:
Please always include a trailing slash.
```suggestion
            help="Upload with or without SSL verification.",
```